src/dbus/server/introspect.xml

```diff
@@ -246,8 +246,16 @@
       <arg name="epskc" type="s" direction="out"/>
     </method>
 
-    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode. -->
+    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode.
+      @retain_active_session: 
+        "false" - Disconnects the Border Agent from any active secure sessions.
+                  Ephemeral key would be cleared after the session is disconnected.
+        "true" - Deactivate ephemeral key mode only when Border Agent state is running
+                 and no active session established. If a commissioner connection is active,
```

Comment:
> Deactivate ephemeral key mode only when Border Agent state is running
                 and no active session established

This is not accurate, the ephemeral key is always deactivated even if the commissioner is connected.

Reply:
Modified accordingly, reopen if not proper. Thanks~